aws/cloudformation/components/database.yml.erb

```diff
@@ -281,6 +329,8 @@
     Properties:
       DBClusterIdentifier: !Sub "${AWS::StackName}-cluster"
       DBClusterParameterGroupName: !Ref AuroraClusterDBParameters
+      # Use same ParameterGroup for writer and all readers so that any reader can be promoted to writer during a Failover.
+      DBInstanceParameterGroupName: !Ref AuroraWriterDBParameters
```

Comment:
Does NOT trigger restart https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-dbinstanceparametergroupname

Reply:
hmmmm ... the instances were provisioned with the default ParameterGroup

<img width="256" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/2157034/041b3028-8a89-4db9-b77c-7917a31d37ba">